src/transactions/tests_helpers.rs

```diff
@@ -627,9 +629,24 @@ pub fn derive_transactions(
         value: 470,
         ..TxOut::default()
     };
+    SpendTransaction::new(
+        vec![spend_unvault_txin.clone()],
+        vec![SpendTxOut::new(dust_txo.clone())],
+        None,
+        &der_cpfp_descriptor,
+        0,
+        true,
+    )
+    .expect_err("Creating a dust output");
+
+    // We can't create a dust change output with the Spend
     SpendTransaction::new(
         vec![spend_unvault_txin],
-        vec![SpendTxOut::Destination(dust_txo.clone())],
+        vec![],
+        Some(DepositTxOut::new(
+            Amount::from_sat(329),
```

Comment:
Technically something just below `DUST_LIMIT` would have been preferable, but this works!